index.js

```diff
@@ -0,0 +1,16 @@
+const productListItem = document.querySelector(".product-list__item");
+const productList = document.querySelector(".product-list");
+const apple = document.createElement(div);
+
+apple.className.add("product-list__title");
```

Comment:
Pozmieniaj `className` na `classList`, bo `className` zwraca jedynie skonatenowanego stringa zawierającego wszystkie nazwy klas, a `classList` to DOM Token List czy coś takiego, który pozwala na ich manipulację.